test/python/test_transport.py

```diff
@@ -143,108 +144,120 @@ def test_add_species_multi(self):
+        visc1_h2 = phase['H2'].species_viscosities[0]
+        mu_poly_h2 = phase.get_viscosity_polynomial(phase.species_index('H2'))
+        phase.set_viscosity_polynomial(phase.species_index('H2'), mu_poly_h2o)
+        visc2_h2 = phase['H2'].species_viscosities[0]
+        phase.set_viscosity_polynomial(phase.species_index('H2'), mu_poly_h2)
+        visc3_h2 = phase['H2'].species_viscosities[0]
+        assert visc1_h2o != visc1_h2
+        assert visc1_h2o == visc2_h2
+        assert visc1_h2 == visc3_h2
+
+    def test_transport_polynomial_fits_conductivity(self, phase):
+        phase.X = {'O2': 1}
+        cond1_o2 = phase.thermal_conductivity
+        lambda_poly_o2 = phase.get_thermal_conductivity_polynomial(phase.species_index("O2"))
+        phase.X = {"H2": 1}
+        cond1_h2 = phase.thermal_conductivity
+        lambda_poly_h2 = phase.get_thermal_conductivity_polynomial(phase.species_index('H2'))
+        phase.set_thermal_conductivity_polynomial(phase.species_index('H2'), lambda_poly_o2)
+        cond2_h2 = phase.thermal_conductivity
+        phase.set_thermal_conductivity_polynomial(phase.species_index('H2'), lambda_poly_h2)
+        cond3_h2 = phase.thermal_conductivity
+        assert cond1_o2 != cond1_h2
+        assert cond1_o2 == cond2_h2
+        assert cond1_h2 == cond3_h2
+
+    def test_transport_polynomial_fits_diffusion(self, phase):
+        D12 = phase.binary_diff_coeffs[1, 2]
+        D23 = phase.binary_diff_coeffs[2, 3]
+        bd_poly_12 = phase.get_binary_diff_coeffs_polynomial(1, 2)
+        bd_poly_23 = phase.get_binary_diff_coeffs_polynomial(2, 3)
+        phase.set_binary_diff_coeffs_polynomial(1, 2, bd_poly_23)
+        phase.set_binary_diff_coeffs_polynomial(2, 3, bd_poly_12)
+        D12mod = phase.binary_diff_coeffs[1, 2]
+        D23mod = phase.binary_diff_coeffs[2, 3]
+        phase.set_binary_diff_coeffs_polynomial(1, 2, bd_poly_12)
+        phase.set_binary_diff_coeffs_polynomial(2, 3, bd_poly_23)
+        D12new = phase.binary_diff_coeffs[1, 2]
+        D23new = phase.binary_diff_coeffs[2, 3]
+        assert D12 != D23
+        assert D12 == D23mod
+        assert D23 == D12mod
+        assert D12 == D12new
+        assert D23 == D23new
+
+
+class TestIonTransport:
+
+    @pytest.fixture(scope='class')
+    def initial_conditions(self):
+        return {
+            'T': 2237,
+            'P': ct.one_atm,
+            'X': 'O2:0.7010, H2O:0.1885, CO2:9.558e-2'
+        }
+
+    @pytest.fixture(scope='function')
+    def gas(self, initial_conditions):
+        gas = ct.Solution('ch4_ion.yaml')
+        gas.TPX = initial_conditions['T'], initial_conditions['P'], initial_conditions['X']
+        return gas
+
+    @pytest.fixture(scope='function')
+    def N2_idx(self, gas):
+        return gas.species_index("N2")
+
+    @pytest.fixture(scope='function')
+    def H3Op_idx(self, gas):
+        return gas.species_index("H3O+")
```

Comment:
I'm not convinced these fixtures are useful. Just accessing these values in the specifc tests below would take the same number of lines of code.

Reply:
You are correct. I think with many of these fixtures, I tried to just translate class attributes into fixtures, and that was the case here. I've moved the values into the tests themselves since there's only a few tests.